Formula/acl2.rb

```diff
@@ -0,0 +1,36 @@
+class Acl2 < Formula
+  desc "The logic and programming language in which you can model computer systems"
+  homepage "https://www.cs.utexas.edu/users/moore/acl2/index.html"
+  url "https://github.com/acl2/acl2/archive/8.3.tar.gz"
+  sha256 "45eedddb36b2eff889f0dba2b96fc7a9b1cf23992fcfdf909bc179f116f2c5ea"
+  license "BSD-3-Clause"
+
+  depends_on "sbcl"
+
+  def install
+    prefix.install Dir["*"]
+    system "make",
+           "-C", prefix,
+           "-j", ENV.make_jobs,
```

Comment:
This is already set by homebrew.

Reply:
Which option you mentioned, `"-C", preifx` or `"-j", ENV.make_jobs`?

Reply:
`-j`

Reply:
I see. I have deleted it.